components/curated-file-table.js

```diff
@@ -0,0 +1,148 @@
+// node_modules
+import { TableCellsIcon } from "@heroicons/react/20/solid";
+import PropTypes from "prop-types";
+// components
+import { BatchDownloadActuator } from "./batch-download";
+import { DataAreaTitle, DataAreaTitleLink } from "./data-area";
+import { FileAccessionAndDownload } from "./file-download";
+import { HostedFilePreview } from "./hosted-file-preview";
+import SortableGrid from "./sortable-grid";
+import Status from "./status";
+// lib
+import { FileSetController } from "../lib/batch-download";
+import { dataSize, truthyOrZero } from "../lib/general";
+
+const filesColumns = [
+  {
+    id: "accession",
+    title: "Accession",
+    display: ({ source }) => (
+      <div className="flex items-start gap-1">
+        <FileAccessionAndDownload file={source} />
+        <HostedFilePreview file={source} buttonSize="sm" />
+      </div>
+    ),
+  },
+  {
+    id: "file_format",
+    title: "File Format",
+    sorter: (item) => item.file_format.toLowerCase(),
+  },
+  {
+    id: "content_type",
+    title: "Content Type",
+    sorter: (item) => item.content_type.toLowerCase(),
+  },
+  {
+    id: "summary",
+    title: "Summary",
+  },
+  {
+    id: "lab",
+    title: "Lab",
+    display: ({ source }) => source.lab?.title,
+    sorter: (item) => (item.lab ? item.lab.title.toLowerCase() : ""),
+  },
+  {
+    id: "file_size",
+    title: "File Size",
+    display: ({ source }) =>
+      truthyOrZero(source.file_size) ? dataSize(source.file_size) : "",
+  },
+  {
+    id: "submitted_file_name",
+    title: "Submitted File Name",
+  },
+  {
+    id: "upload_status",
+    title: "Upload Status",
+    display: ({ source }) => <Status status={source.upload_status} />,
+  },
+];
+
+/**
+ * Display a sortable table of the given files.
+ */
+export default function CuratedFileTable({
```

Comment:
Could you make this a non-default export, and then import it with `import { CuratedFileTable } from "../../components/curated-file-table";`? I’d like to make all new components non-default exports.